TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/TestTele.java

```diff
@@ -56,8 +56,7 @@ public void runOpMode() {
             robot.calculateDrivePowers(
```

Comment:
move main tele code to new MainTele opmode

Reply:
not sure if this is what you wanted but I just renamed TestTele to MainTele